sth
descriptor 'reshape' for 'numpy.ndarray' objects doesn't apply to a 'tuple' object, Traceback (most recent call last):
  File "/tmp/ipykernel_180895/277800734.py", line 8, in _newPdOr
    try: return oldPdOr(self, v)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/ops/common.py", line 72, in new_method
    return method(self, other)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/arraylike.py", line 80, in __or__
    return self._logical_method(other, operator.or_)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/series.py", line 6254, in _logical_method
    res_values = ops.logical_op(lvalues, rvalues, op)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/ops/array_ops.py", line 395, in logical_op
    res_values = na_logical_op(lvalues, rvalues, op)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/ops/array_ops.py", line 333, in na_logical_op
    return result.reshape(x.shape)
TypeError: descriptor 'reshape' for 'numpy.ndarray' objects doesn't apply to a 'tuple' object

descriptor 'reshape' for 'numpy.ndarray' objects doesn't apply to a 'tuple' object, Traceback (most recent call last):
  File "/tmp/ipykernel_181692/505019376.py", line 8, in _newPdOr
    try: return oldPdOr(self, v)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/ops/common.py", line 72, in new_method
    return method(self, other)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/arraylike.py", line 80, in __or__
    return self._logical_method(other, operator.or_)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/series.py", line 6254, in _logical_method
    res_values = ops.logical_op(lvalues, rvalues, op)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/ops/array_ops.py", line 395, in logical_op
    res_values = na_logical_op(lvalues, rvalues, op)
  File "/home/kelvin/anaconda3/envs/torch/lib/python3.9/site-packages/pandas/core/ops/array_ops.py", line 333, in na_logical_op
    return result.reshape(x.shape)
TypeError: descriptor 'reshape' for 'numpy.ndarray' objects doesn't apply to a 'tuple' object

